trustar_consts.py

```diff
@@ -48,7 +48,6 @@
 TRUSTAR_TIME_FORMAT = "%Y-%m-%dT%H:%M:%S%z"
 TRUSTAR_ERR_TIME_FORMAT = "Timestamp format incorrect"
 TRUSTAR_ERR_TIME_PARSE = "Could not parse given time: {value_error}"
-TRUSTAR_PRIORITY_EVENT_SCORES = [-1, 0, 1, 2, 3]
```

Comment:
why did we move this out of the constants file? if it's because a list is mutable, then changing it to a tuple should fix it, i.e.

```
TRUSTAR_PRIORITY_EVENT_SCORES = (-1, 0, 1, 2, 3)
```

Reply:
oh I see the commit message in https://github.com/splunk-soar-connectors/trustar/pull/4/commits/12e49507171fe90c91555688810100ad38a31b97

I thought you had discovered the fix for the AST parsing issue previously? or was that a different issue? :)

Reply:
-1 is a negative value and gets parsed into a more complex strucure in ast

```
>>> print(ast.dump(ast.parse('[-1,0,1]', mode='eval'), indent=4))
Expression(
    body=List(
        elts=[
            UnaryOp(
                op=USub(),
                operand=Constant(value=1)),
            Constant(value=0),
            Constant(value=1)],
        ctx=Load()))
```